variables.tf

```diff
@@ -444,5 +444,12 @@ variable "repositories" {
       teams       = ["infrastructure"]
       topics      = ["mage-os", "magento", "ecommerce", "magento2", "adobecommerce"]
     }
+
+    magento-cache-clean = {
```

Comment:
Sorry for the nitpicking, but don't we have the repositories in here sorted alphabetically? :see_no_evil:

Reply:
OH, ok!

Reply:
Should be better  now